no-browser-site-scrapers/Albertsons/config.js

```diff
@@ -0,0 +1,10 @@
+const site = {
+    name: "Albertsons",
+    website: "https://www.mhealthappointments.com/covidappt",
+    nationalStoresJson:
+        "https://s3-us-west-2.amazonaws.com/mhc.cdn.content/vaccineAvailability.json?v=1617818069756",
```

Comment:
You may want to make the `?v=1617818069756` dynamic eg `?v={$UNIX_TIMESTAMP}`. The ?v= is probably being used to cache bust the resource and by putting a new number you ensure a fresh request to the origin.

Reply:
Thanks! I'll change that.